cloud/monitoring/anomaly-detection.md

```diff
@@ -0,0 +1,213 @@
+
+1. On the **Monitoring > Resources Status** page, click the Anomaly Detection service you want, and then click the **Graph** tab in the details panel.
+2. Click the **Edit anomaly detection data** icon (wrench) at the top right of the tab. The graph opens in a pop-up window.
+3. Use the slider in the **Manage envelope size** section to change the range of the predictions. In the preview, checks outside the envelope are shown by red dots.
+4. Click **Save**. The new envelope size is applied from that moment on. The envelope that was already calculated stays the same (this means that the changes are not visible immediately on the graph).
+
+### Step 3: Activate status changes
+
+If, by regularly checking the generated model and the data from the
+**Monitoring > Performances > Graphs** menu, you think that your model is
+stable, you can activate status changes. Once you enable this option, the status of the service will switch to CRITICAL [SOFT](../alerts-notifications/concepts.md#status-types) as soon as the metric goes below the lower threshold or above the upper threshold.
+
+1. Go to the **Configuration > Services > Anomaly Detection** menu and click the Anomaly Detection service you want.
+
+2. In the **Alerting options** section, enable the **Enable change of status** option.
+
+3. In the **Detect anomalies after** field, enter the number of deviations before the service switches to a CRITICAL [HARD](../alerts-notifications/concepts.md#status-types) state. (When enabled, notifications are only sent when the service switches from SOFT to HARD.)
+
+4. Click **Save**.
+
+5. [Deploy the configuration](monitoring-servers/deploying-a-configuration.md).
+
+### Activate the notification process
+
+When you are satisfied that the status changes you see are relevant (they do happen when an incident starts or finishes), then your Anomaly Detection service is fully operational. You can then activate the notifications.
+
+1. Go to **Configuration > Services > Anomaly Detection** and click the Anomaly Detection service you want.
+
+2. Complete the following fields:
+
+  - **Enable notification**: select **Enabled**.
+  - **Implied Contacts**: select who will receive notifications for this service.
+  - **Implied Contact Groups**: select the contact groups that will receive notifications for this service.
+  - **Notification Interval**: define how frequently notifications should be sent once the service has entered a CRITICAL HARD state and has not been acknowledged yet. The default value is **0**, which means only one notification will be sent per status change.
+  - **Notification Period**: select the time period during which these users may receive notifications.
+  - **Notification Type**: select the types of notifications you want to receive (when the service enters a CRITICAL state and/or when it goes back to normal).
+
+3. Click **Save**.
+
```

Comment:
That section should be commpletely changed.
Notification on cloud works differently. 
We can make a generic statement that Anomaly Detection services are not different from other services notification wise.